README.md

```diff
@@ -18,10 +18,12 @@ $ docker run -it --rm relbot ...command...
 
 ### Development
 
-```
-$ python3 -m venv env
-$ pip install -r requirements.txt
-$ pip install pytest
+```sh
+python3 -m venv env
+source env/bin/activate
+pip install -r requirements.txt
+pip install pytest pre-commit
+pre-commit install --install-hooks
```

Comment:
Just for my own lack of understanding. How do we run the linter manually?

Reply:
`pre-commit run --all-files` 🙂  https://pre-commit.com/#4-optional-run-against-all-the-files